test/Core.Test/AutoFixture/OrganizationFixtures.cs

```diff
@@ -33,7 +33,9 @@ public void Customize(IFixture fixture)
 
             fixture.Customize<Core.Models.Table.Collection>(composer =>
                 composer
-                    .With(c => c.OrganizationId, organizationId));
+                    .With(c => c.OrganizationId, organizationId)
+                    .Without(o => o.CreationDate)
+                    .Without(o => o.RevisionDate));
```

Comment:
Had to change this after modifying CreationDate and RevisionDate to be public.

Reply:
should be able to revert this if we go with the pure function idea